src/Bicep.Decompiler.IntegrationTests/Files/Working/101-1vm-2nics-2subnets-1vnet/azuredeploy.bicep

```diff
@@ -18,25 +18,25 @@ param storageAccountType string = 'Standard_LRS'
 @description('Location for all resources.')
 param location string = resourceGroup().location
 
-var virtualMachineName_var = 'VM-MultiNic'
+var virtualMachineName = 'VM-MultiNic'
 var nic1_var = 'nic-1'
 var nic2_var = 'nic-2'
-var virtualNetworkName_var = 'virtualNetwork'
+var virtualNetworkName = 'virtualNetwork'
 var subnet1Name = 'subnet-1'
 var subnet2Name = 'subnet-2'
-var publicIPAddressName_var = 'publicIp'
-var subnet1Ref = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName_var, subnet1Name)
-var subnet2Ref = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName_var, subnet2Name)
-var diagStorageAccountName_var = 'diags${uniqueString(resourceGroup().id)}'
-var networkSecurityGroupName_var = 'NSG'
-var networkSecurityGroupName2_var = '${subnet2Name}-nsg'
+var publicIPAddressName = 'publicIp'
+var subnet1Ref = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, subnet1Name)
+var subnet2Ref = resourceId('Microsoft.Network/virtualNetworks/subnets', virtualNetworkName, subnet2Name)
+var diagStorageAccountName = 'diags${uniqueString(resourceGroup().id)}'
+var networkSecurityGroupName = 'NSG'
+var networkSecurityGroupName2 = '${subnet2Name}-nsg'
 
-resource virtualMachineName 'Microsoft.Compute/virtualMachines@2019-12-01' = {
-  name: virtualMachineName_var
+resource virtualMachine 'Microsoft.Compute/virtualMachines@2019-12-01' = {
```

Comment:
Nice that this also addresses some occurrences of `_var`!

Reply:
Yep, gets rid of the resource vs resource name var conflict.  Removed 80% of "_var"s and 90% of "_resource" from quickstarts.  May not need anything else, but of course quickstarts is not the average template.  

I'd would like to implement the rule to use symbolic names when possible, though, which could allow us to remove variables used only in resource names, even if not using the "*Name" pattern.